static/app/components/alerts/snoozeAlert.tsx

```diff
@@ -0,0 +1,103 @@
+import styled from '@emotion/styled';
+
+import {Client} from 'sentry/api';
+import {Button} from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import {DropdownMenu, MenuItemProps} from 'sentry/components/dropdownMenu';
+import {IconChevron, IconMute, IconSound} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {Organization} from 'sentry/types';
+import withApi from 'sentry/utils/withApi';
+import withOrganization from 'sentry/utils/withOrganization';
+
+type Props = {
+  api: Client;
+  isSnoozed: boolean;
+  organization: Organization;
+  projectId: string;
```

Comment:
no more slugs as ids
```suggestion
  projectSlug: string;
```